Scripts/script-CommonServer.yml

```diff
@@ -1839,6 +1839,19 @@ script: |-
 
       /**************************** HMAC_SHA256 end *******************************/
 
+    /**************************** REGEX FORMATTING *******************************/
+
+      var ipRegex = /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/;
+      var emailRegex = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
```

Comment:
You should use the regex from the ExtractEmail script, and change the ExtractEmail to use this regex...